database/snapshotOptions/templateExport.json

```diff
@@ -30,5 +30,6 @@
   "excludeInsertScripts": [],
   "includePrefs": false,
   "includeLocalisation": false,
-  "resetFiles": false
+  "resetFiles": false,
+  "templates": { "checkVersionOnImport": true }
```

Comment:
An additional option to tell the template snapshot import to check for existing templates with the same versionId, and return and error if there is (so you can't accidentally import two copies of the same version)